app/views/admin/_adminheader.html.erb

```diff
@@ -18,6 +21,9 @@
         <li class="nav-item">
           <%= link_to "Orders (#{Order.not_delivered.count})", admin_orders_path, class: "nav-link bg-gradient btn-success" %>
         </li>
+        <li class="nav-item">
+          <%= link_to "Add Kudos", admin_employee_kudos_for_all_path(Employee), class: "nav-link bg-gradient btn-success" %>
```

Comment:
is the `Employee` parameter necessary here?

Reply:
I changed it